.github/actions/code-coverage/action.yml

```diff
@@ -0,0 +1,53 @@
+name: On-chain Code Coverage
+description: Runs code coverage evaluation for on-chain project.
+inputs:
+  alchemy_eth_api:
+    description: The Alchemy API key to use for Ethereum RPC calls.
```

Comment:
These are actually RPC URIs (rather than just keys) correct? We should update the description (if not the parameter name) to match.